g-sgov/g-sgov-aktuality/g-sgov-aktuality-glosář-MaNe.ttl

```diff
@@ -60,31 +90,71 @@ g-sgov-aktuality:glosář
+                "Hodnota této vlastnosti je z Číselníku pro témata událostí publikovaného jako číselník v Národním katalogu otevřených dat."@cs ;
+        <http://www.w3.org/2004/02/skos/core#broader>
+                <https://slovník.gov.cz/základní/pojem/vztah> , <https://slovník.gov.cz/základní/pojem/typ-vztahu> ;
+        <http://www.w3.org/2004/02/skos/core#hiddenLabel>
+                "Téma"@cs ;
+        <http://www.w3.org/2004/02/skos/core#inScheme>
+                g-sgov-aktuality:glosář ;
+        <http://www.w3.org/2004/02/skos/core#prefLabel>
+                "má téma"@cs .
+
+g-sgov-aktuality-pojem:popis
+        a                <http://www.w3.org/2004/02/skos/core#Concept> ;
+        rdfs:subClassOf  <http://purl.org/dc/terms/description> ;
+        <http://purl.org/dc/terms/description>
+                "Popis aktuality"@cs ;
+        <http://www.w3.org/2004/02/skos/core#broader>
+                <https://slovník.gov.cz/základní/pojem/vlastnost> , <https://slovník.gov.cz/základní/pojem/typ-vlastnosti> ;
+        <http://www.w3.org/2004/02/skos/core#inScheme>
+                g-sgov-aktuality:glosář ;
+        <http://www.w3.org/2004/02/skos/core#prefLabel>
+                "Popis"@cs .
+
+g-sgov-aktuality-pojem:má-typ
+        a       <http://www.w3.org/2004/02/skos/core#Concept> ;
+        <http://purl.org/dc/terms/description>
+                "Hodnota této vlastnosti je z Číselníku pro typy událostí publikovaného jako číselník v Národním katalogu otevřených dat."@cs ;
+        <http://www.w3.org/2004/02/skos/core#broader>
+                <https://slovník.gov.cz/základní/pojem/vztah> , <https://slovník.gov.cz/základní/pojem/typ-vztahu> ;
+        <http://www.w3.org/2004/02/skos/core#hiddenLabel>
+                "Typ"@cs ;
+        <http://www.w3.org/2004/02/skos/core#inScheme>
+                g-sgov-aktuality:glosář ;
+        <http://www.w3.org/2004/02/skos/core#prefLabel>
+                "má typ"@cs .
+
+g-sgov-aktuality-pojem:relevantní-do
+        a       <http://www.w3.org/2004/02/skos/core#Concept> ;
+        <http://purl.org/dc/terms/description>
+                "Časový okamžik do kdy je aktualita relevantní."@cs ;
         <http://www.w3.org/2004/02/skos/core#broader>
                 <https://slovník.gov.cz/základní/pojem/vztah> , <https://slovník.gov.cz/základní/pojem/typ-vztahu> ;
+        <http://www.w3.org/2004/02/skos/core#hiddenLabel>
+                "Relevantní do"@cs ;
         <http://www.w3.org/2004/02/skos/core#inScheme>
                 g-sgov-aktuality:glosář ;
         <http://www.w3.org/2004/02/skos/core#prefLabel>
-                "má téma"@cs ;
-        <http://www.w3.org/2004/02/skos/core#scopeNote>
-                "Hodnota této vlastnosti je z Číselníku pro témata událostí publikovaného jako číselník v Národním katalogu otevřených dat."@cs .
+                "relevantní do"@cs .
 
-g-sgov-aktuality-pojem:má-oznamovatele
+g-sgov-aktuality-pojem:má-pořadatele
```

Comment:
Proč má-pořadatele? Má tam být má-oznamovatele, jak jsem měl já.

Reply:
Podle mne mas nejaky aktualnejsi model nez ja.

Kdyz to generuji z Tveho modelu tak mi to vygeneruje ":má-pořadatele". Vychazim z:
https://drive.google.com/file/d/1MBdYeRzIv3Ksy0HhF6QIcnLcA8JIvee4/view
Relevantni Trello ticket:
https://trello.com/c/JPapczHh/120-namodelovat-4-typ%C5%AF-slovn%C3%ADk%C5%AF-pro-ofn

Jinak tady bych doporucoval, nepouzivat jiny skos:prefLabel od jmena prvku v modelio modeli. Aktualne modelio funguje tak, ze kdyz editujes jmeno prvku v modeli automaticky se podle toho vyplni skos:prefLabel. Nevidim duvod proc by jmeno prvku nemelo odpovidat skos:prefLabel-u, kdyz oboje je v cestine. Pro alternativne jmena mame predsa skos:altLabel.
![Selection_148](https://user-images.githubusercontent.com/3962237/91225561-1b3c5880-e724-11ea-9562-ca4141fcd23a.jpg)